open_alchemy/helpers/peek.py

```diff
@@ -854,10 +854,15 @@ def _peek_key(
     """Execute peek_key."""
     _check_schema_schemas_dict(schema, schemas)
 
+    # Define valid OpenAlchemy prefixes.
+    valid_prefixes = ["x-", "x-openalchemy-"]
```

Comment:
I know this isn't the final code yet, one thought that came to mind, this might be a good constant to define outside the function scope